### PR TITLE
GOV-165 Stop computing ratings for views in SonarQube

### PR DESCRIPTION
**Pull request by teryk**

### CODE REVIEW

File: it/it-plugins/access-secured-props-plugin/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.sonarsource.sonarqube</groupId>
     <artifactId>it-plugins</artifactId>
-    <version>6.2-SNAPSHOT</version>
+    <version>6.3-SNAPSHOT</version>
```
Comment (nixel2007) [inline it/it-plugins/access-secured-props-plugin/pom.xml]: 6.3?

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: it/it-plugins/access-secured-props-plugin/pom.xml (hunk 1 above)
Comment (julienlancelot) [inline it/it-plugins/access-secured-props-plugin/pom.xml]: It should be 6.2

Comment (teryk) [inline it/it-plugins/access-secured-props-plugin/pom.xml]: thanks nixel, udpated

**Pull request merged**